src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.navv {
+  display: flex;
+  align-items: stretch;
+  flex-direction: row;
```

Comment:
it's default value

Reply:
.navv {
  display: flex;
  align-items: stretch;
  flex-direction: row; 

do I  need to delete it?